app/i18n/messages/en.json

```diff
@@ -308,6 +308,15 @@
   "ManageReservationsList.statusHeader": "Status",
   "ManageReservationsList.pinHeader": "PIN",
   "ManageReservationsList.actionsHeader": "Select action",
+  "ManageReservationsFilters.searchLabel": "Search",
```

Comment:
Should we have duplicated string like this for example, re-use them instead?
We have like 2 - 3 `Search` already

Reply:
No, absolutely not. It doesn't matter if there are identical translations. 
It's the string/translation for "ManageReservationsFilters.searchLabel", and it can change. It might be same as some other string/translation, or it might not.